budget/_vars-budget.html

```diff
@@ -3,7 +3,7 @@
 {% set nav_items = [
     (path, 'index', 'Budget and Strategy'),
     (path + 'strategic-plan/', 'strategic-plan', 'Strategic Plan'),
-    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan Report'),
-    (path + 'financial-report/', 'financial-report', 'Financial Report & Update'),
-    (path + 'funding-request/', 'funding-request', 'Funding Request'),
+    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan & Report'),
```

Comment:
> Performance Plan and Report is actually a single document, released annually. 

Each year isn't considered a separate document? E.g. the 2015 Performance Plan and Report, the 2014 Performance Plan and Report, etc. They are separate downloads on the page that links to.